source/backend/core/Extensions/EnumExtensions.cs

```diff
@@ -1,4 +1,6 @@
+using Pims.Core.Security;
```

Comment:
nit: github is complaining here about linting. perhaps need to sort the using statements